packages/client/test/rpc/eth/getBalance.spec.ts

```diff
@@ -59,53 +56,48 @@ describe(
 
       // verify balance is genesis amount minus amountSpent
       const expectedNewBalance = genesisBalance - amountSpent
-      req = params(method, [address.toString(), 'latest'])
-      expectRes = (res: any) => {
-        const msg = 'should return the correct balance after a tx'
-        assert.equal(res.body.result, bigIntToHex(expectedNewBalance), msg)
-      }
-      await baseRequest(server, req, 200, expectRes, false)
+      res = await rpc.request(method, [address.toString(), 'latest'])
+
+      msg = 'should return the correct balance after a tx'
+      assert.equal(res.result, bigIntToHex(expectedNewBalance), msg)
 
       // verify we can query with "earliest"
-      req = params(method, [address.toString(), 'earliest'])
-      expectRes = (res: any) => {
-        const msg = "should return the correct balance with 'earliest'"
-        assert.equal(res.body.result, bigIntToHex(genesisBalance), msg)
-      }
-      await baseRequest(server, req, 200, expectRes, false)
+      res = await rpc.request(method, [address.toString(), 'earliest'])
+
+      msg = "should return the correct balance with 'earliest'"
+      assert.equal(res.result, bigIntToHex(genesisBalance), msg)
```

Comment:
minor nit, but unless the msgs are reused, I'd personally just in-line them rather than declaring them as a separate variable.